src/display-fibonacci-sequence.js

```diff
@@ -0,0 +1,21 @@
+function displayFibonacci(number) {
```

Comment:
we are working on `find the largest number in a sequence of n numbers` task so my question is why we need fibonacci in this ticket ? 
So pls remove this due to unrelated code